packages/parser-typescript-config/src/schema/tsConfigSchema.json

```diff
@@ -0,0 +1,473 @@
+{
```

Comment:
@sarvaje, @molant l presume the release script should also update this file?

Reply:
This file is not exactly a copy/past of the original one. I'm adding a couple of "additionalProperties": false.